code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm

```diff
@@ -205,12 +224,49 @@ Difficulty: Very Hard
 		var/turf/E = get_step(src, d)
 		shoot_projectile(E)
 
-/mob/living/simple_animal/hostile/megafauna/colossus/proc/telegraph()
+/mob/living/simple_animal/hostile/megafauna/colossus/proc/telegraph(mode)
 	for(var/mob/M in range(10, src))
 		M.flash_screen_color("#C80000", 1)
 		shake_camera(M, 4, 3)
 	playsound(src, 'sound/magic/narsie_attack.ogg', 200, TRUE)
+	if(mode)
+		say("<span class='colossus'>\"<b>[mode]</b>\"</span>", sanitize = FALSE, ignore_speech_problems = TRUE, ignore_atmospherics = TRUE)
+
+/mob/living/simple_animal/hostile/megafauna/colossus/proc/final_attack()
+	icon_state = "eva_attack"
+	say("<span class='colossus'>\"<b>PERISH MORTAL!</b>\"</span>", sanitize = FALSE, ignore_speech_problems = TRUE, ignore_atmospherics = TRUE)
+	telegraph()
+	ranged_cooldown += 2000 //YYeah let us NOT have people get tripple attacked
+	SLEEP_CHECK_DEATH(2.5 SECONDS) //run
 
+	var/finale_counter = 10
+	for(var/i in 1 to 20)
+		if(finale_counter > 4)
+			telegraph()
+			blast(do_sleep = FALSE)
+
+		if(finale_counter > 1)
+			finale_counter -= 1
+
+		var/turf/start_turf = get_turf(src)
+		for(var/turf/target_turf in RANGE_TURFS(12, start_turf))
+			if(prob(min(finale_counter, 2)) && target_turf != get_turf(src))
+				shoot_projectile(target_turf)
+
+		SLEEP_CHECK_DEATH(finale_counter + 0.2 SECONDS) //Doubled from TG, this was insane
+
+	for(var/i in 1 to 3)
+		telegraph()
+		random_shots(do_sleep = FALSE)
+		finale_counter += 6
+		SLEEP_CHECK_DEATH(finale_counter)
+
+	for(var/i in 1 to 3)
+		telegraph()
+		dir_shots()
+		SLEEP_CHECK_DEATH(1 SECONDS)
+	icon_state = initial(icon_state)
+	ranged_cooldown = world.time += 40
```

Comment:
```suggestion
	ranged_cooldown = world.time + 4 SECONDS
```
I think this is what you meant. I dont think colossus is supposed to increment the world.time

Reply:
hahaha yeah that might be bad